src/single-movie/SingleMovie.test.js

```diff
@@ -1,68 +1,105 @@
 import React from 'react';
-import Movies from '../movies/Movies.js';
 import SingleMovie from './SingleMovie'
-import { render, waitFor, screen, getByText,
-  getByAltText, getByTitle} from '@testing-library/react'
+import { render, waitFor, screen} from '@testing-library/react'
 import '@testing-library/jest-dom'
 import { MemoryRouter } from 'react-router-dom'
 import  { getSingleMovie }  from '../apiCalls.js'
 jest.mock('../apiCalls.js')
 
+const ratings = [{
+  created_at: "2020-10-15T21:31:06.428Z",
+  id: 2887,
+  movie_id: 613504,
+  rating: 4,
+  updated_at: "2020-10-15T21:31:06.428Z",
+  user_id: 80
+}];
```

Comment:
Smart thinking making the ratings a global variable because we need it in a few spots